### PR TITLE
Setup Codespaces .devcontainer

### PR DESCRIPTION
**Pull request by simon-curtis**
I have added a simple .devcontainer setup to help with the initial setup of codespaces.

I would be nice also to have either a `gleam tool add wasm` or a feature config to include wasm compiler in the feature image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,23 @@
...
+
+```sh
+# Running the project will generate a static site under the `public/` folder
 gleam run
 
-# It's now all the in `public/` directory
-```
+# Running the server will serve the newly created static files
+http-server public/
+```
```
Comment (lpil) [inline README.md:24]: Could you revert the changes to this file please? Thank you.

Comment (simon-curtis) [reply README.md]: Done :)

**Review by lpil — APPROVED** [review]
Thank you
